content/podcasts/index.md

```diff
@@ -2,5 +2,10 @@
 title = "Podcasts"
 template = "podcasts.html"
 aliases = ["/matrixlive/"]
+extra.meta_description = """
+Matrix Live is the weekly show produced by the Matrix Foundation to talk about
+the latest Matrix news. Open Tech Will Save Us (on pause) was a show to talk
```

Comment:
Similar to the other extra text "(on pause)" feels weird here imho. Also not sure how often google updates these. So it might cause issues down the line.